src/App.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import { ThemeProvider } from 'styled-components'
-import { theme } from 'styles/global'
+import { theme } from './styles/global'
+import Layout from './layout/default'
```

Comment:
```suggestion
import Layout from './layout/Layout'
```

這個檔名 `default` 是不是在搞???
可以取一樣的嗎問號